neovim/msgpack_rpc/event_loop/uv.py

```diff
@@ -73,12 +73,11 @@ def _connect_child(self, argv):
                             flags=pyuv.UV_CREATE_PIPE + pyuv.UV_WRITABLE_PIPE)
         stderr = pyuv.StdIO(self._error_stream,
                             flags=pyuv.UV_CREATE_PIPE + pyuv.UV_WRITABLE_PIPE)
-        self._process = pyuv.Process(self._loop)
-        self._process.spawn(file=argv[0],
-                            exit_callback=self._on_exit,
-                            args=argv[1:],
-                            flags=pyuv.UV_PROCESS_WINDOWS_HIDE,
-                            stdio=(stdin, stdout, stderr,))
+        pyuv.Process.spawn(self._loop,
+                           args=argv,
+                           exit_callback=self._on_exit,
+                           flags=pyuv.UV_PROCESS_WINDOWS_HIDE,
+                           stdio=(stdin, stdout, stderr,))
```

Comment:
Ok, so we should require versions newer than 1.0 on Windows.

About Linux, what is your idea? Support both API versions?
I think we should just raise an exception or use just ignore pyuv and use asyncio if pyuv is < 1.0.0.